src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -406,24 +504,25 @@ private async Task<DownloadedBuild> GatherDropForBuildAsync(Build build, string
             // If the drop is separated, calculate the directory name based on the last element of the build
             // repo uri plus the build number (to disambiguate overlapping builds)
             string outputDirectory = rootOutputDirectory;
+            string repoUri = build.AzureDevOpsRepository ?? build.GitHubRepository;
```

Comment:
>build.AzureDevOpsRepository ?? build.GitHubRepository; [](start = 29, length = 54)

Is it warrantied that either one of these is set or is there a chance that repoUri could be null/empty? If so, we should add a check so we don't NullRef

Reply:
One will be set.